rust/kernel/print.rs

```diff
@@ -147,12 +147,14 @@ macro_rules! print_macro (
         // ones. All `__LOG_PREFIX`s are null-terminated as they are generated
         // by the `module!` proc macro or fixed values defined in a kernel
         // crate.
-        unsafe {
-            $crate::print::call_printk(
-                &$format_string,
-                crate::__LOG_PREFIX,
-                format_args!($($arg)+),
-            );
+        match format_args!($($arg)+) {
+            args => unsafe {
+                $crate::print::call_printk(
+                    &$format_string,
+                    crate::__LOG_PREFIX,
+                    args,
+                );
+            }
```

Comment:
This fix has been merged upstream in https://lore.kernel.org/rust-for-linux/167461338342.26446.17308850166282928589.pr-tracker-bot@kernel.org/T/#t already.

Reply:
That's great! I'll drop this one then. Sorry for being so late to bringing this up (I noticed quite a while ago but it had been sitting in my branch...)